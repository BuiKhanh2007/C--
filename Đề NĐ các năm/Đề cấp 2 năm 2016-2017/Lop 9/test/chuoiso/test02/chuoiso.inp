4571 47
14
85
93
48
96
4
67
61
36
15
44
53
21
43
98
18
22
83
98
81
47
25
76
61
90
23
27
59
33
61
52
38
49
9
97
31
39
95
90
80
55
47
68
76
14
65
28
