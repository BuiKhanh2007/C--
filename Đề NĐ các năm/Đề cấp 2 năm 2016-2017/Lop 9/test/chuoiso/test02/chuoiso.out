0
6
1
1
5
1
9
1
1
5
4
0
8
1
3
5
1
8
3
2
9
4
1
1
7
5
1
1
7
1
1
8
7
1
1
1
1
1
7
1
4
9
1
1
0
1
7
